### PR TITLE
Add one enum member to cloudPcOnPremisesConnectionHealthCheckErrorType

### PR DESCRIPTION
**Pull request by RuiHou105**

### CODE REVIEW

File: api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ The result of a cloud PC on-premises connection health check.
 |adJoinCheckIncorrectCredentials|The active domain join check failed because the domain credentials are incorrect. Please update the username and password.|
 |adJoinCheckOrganizationalUnitNotFound|The active domain join check failed because the specified organizational unit was not found. Please re-enter organization unit.|
 |adJoinCheckOrganizationalUnitIncorrectFormat|The active domain join check failed beccause the format of the specified organizational unit is incorrect. Example format: “OU=OU1,OU=OU2,OU=OU3,DC=DC1”.|
+|adJoinCheckComputerObjectAlreadyExists|The computer object required to check AD domain join is already in another Cloud PC Organizational Unit. Please move the computer object from the old organizational unit to the connection specified organizational unit path.|
```
Comment (Lauragra) [inline api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md:48]: Active Directory should spelled out and not abbreviated AD. Also, I don't think Cloud PC Organizational Unit should be title capped.

**Review by AshleyYangSZ — APPROVED** [review]

**Comment by RuiHou105** [comment]
1. Related api.md: https://microsoftgraph.visualstudio.com/_git/onboarding/pullrequest/7576

2.Related CSDL (schema-Prod-beta.csdl): https://microsoftgraph.visualstudio.com/onboarding/_git/AGS-OnboardingAutomationPipeline/pullrequest/8316

**Review by cohanley — APPROVED** [review]

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ The result of a cloud PC on-premises connection health check.
 |adJoinCheckIncorrectCredentials|The active domain join check failed because the domain credentials are incorrect. Please update the username and password.|
 |adJoinCheckOrganizationalUnitNotFound|The active domain join check failed because the specified organizational unit was not found. Please re-enter organization unit.|
 |adJoinCheckOrganizationalUnitIncorrectFormat|The active domain join check failed because the format of the specified organizational unit is incorrect. Example format: “OU=OU1,OU=OU2,OU=OU3,DC=DC1”.|
+|adJoinCheckComputerObjectAlreadyExists|While attempting domain join, the computer account already exists in the domain. The computer account can't be found in the Organizational Unit provided in the on-premises network connection while the computer name already exists in the domain. This is often caused by moving the computer object out of the OU configured in the on-premises network connection. Please move the computer object back to the target OU.|
```
[the remark this answers is not in the document]
Comment (RuiHou105) [reply api-reference/beta/resources/cloudpconpremisesconnectionhealthcheck.md:48]: Corrected. Thanks.

**Comment by RuiHou105** [comment]
@JarbasHorst I've resolved all your comments. Could you please review it again? Thanks.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**